Rdmp.Core/CommandLine/Runners/DleRunner.cs

```diff
@@ -5,7 +5,10 @@
 // You should have received a copy of the GNU General Public License along with RDMP. If not, see <https://www.gnu.org/licenses/>.
 
 using System;
+using System.Collections.Generic;
+using System.IO;
 using System.Linq;
+using Org.BouncyCastle.Security.Certificates;
```

Comment:
Is the BouncyCastle using actually needed? It tends to sneak in by accident for some reason...

Reply:
Nope, must have accidentally imported it (+ in a few other places)